core/src/main/java/org/apache/iceberg/jdbc/JdbcCatalog.java

```diff
@@ -250,10 +266,111 @@ public void setConf(Configuration conf) {
     this.conf = conf;
   }
 
+  private boolean insertProperties(Namespace namespace, Map<String, String> properties) {
+    String namespaceName = JdbcUtil.namespaceToString(namespace);
+
+    try {
+      int insertedRecords = connections.run(conn -> {
+        StringBuilder sqlStatement = new StringBuilder(JdbcUtil.INSERT_NAMESPACE_PROPERTIES_SQL);
+
+        for (int i = 0; i < properties.size() - 1; i++) {
+          sqlStatement.append(", " + JdbcUtil.INSERT_PROPERTIES_VALUES_BASE);
```

Comment:
Nit: Since you're already using the string builder, don't concatenate the strings. Just call `.append` twice. Once for the comma, and once for the values placeholders.

Reply:
Agreed. Will do.